autorel/clients/github_client.py

```diff
@@ -15,46 +15,46 @@
 
 class Github:
 
-    '''
+    """
         Client interacting with the Github REST API
-    '''
+    """
 
     def __init__(self, endpoint, port, token_or_login, password, clientID,
                  clientSecret, userAgent, timeout):
-        self._authorizationHeaders = {}
+        self._headers = {}  # HTTP headers
```

Comment:
Why is it a class member? I mean, `_headers` is something that belongs to HTTPClient, not to GitHub.